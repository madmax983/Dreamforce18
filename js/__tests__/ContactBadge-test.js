import React from 'react';
import ContactBadge from '../ContactBadge.js';

import renderer from 'react-test-renderer';

test('renders correctly', () => {
    const contact = {
        FirstName: "Obi Wan",
        LastName: "Kenobi",
        Title: "Jedi Master",
        MobilePhone: "(999) 999-9999",
        Department: "The Force",
        Email: "obiwan@moseisley.net",
        HomePhone: "(111) 111-1111"
    };
    const tree = renderer.create(<ContactBadge contact={contact} />).toJSON();
    expect(tree).toMatchSnapshot();
});