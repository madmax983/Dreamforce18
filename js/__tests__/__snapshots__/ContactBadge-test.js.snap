// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#1abc9c",
        "borderRadius": 25,
        "height": 50,
        "justifyContent": "center",
        "marginRight": 5,
        "width": 50,
      },
      Object {
        "backgroundColor": "#e67e22",
      },
    ]
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "backgroundColor": "transparent",
        "color": "white",
        "fontFamily": "Helvetica Neue",
        "fontSize": 19,
      }
    }
  >
    OK
  </Text>
</View>
`;
